lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -339,7 +339,7 @@ svi_autostate:
   kind: boolean
   get_value: '/^(?:no )?autostate$/'
   set_value: "<state> autostate"
-  default_value: true
+  default_value: false
```

Comment:
This one has been difficult to pin down and I think it has changed several times now. Please test this on every supported platform.